apps/examples/src/examples/toasts-and-dialogs/ToastsDialogsExample.tsx

```diff
@@ -0,0 +1,57 @@
+import { ExampleDialog, TLComponents, Tldraw, useDialogs, useToasts } from 'tldraw'
```

Comment:
Kind of strange that we export the ExampleDialog and other debug menu items. Not sure if we should be doing this. In this example, let's create a new Dialog rather than use the ExampleDialog one, so that we can show what that looks like.

Reply:
Will do!